api/v1alpha1/nodeobservabilitymachineconfig_helpers.go

```diff
@@ -20,6 +20,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// UpdateLastReconcileTime is for updating LastReconcile in NodeObservabilityMachineConfigStatus
+func UpdateLastReconcileTime(status *NodeObservabilityMachineConfigStatus) {
```

Comment:
How about binding the function to become a method of `NodeObservabilityMachineConfigStatus`?
```suggestion
func (status *NodeObservabilityMachineConfigStatus) UpdateLastReconcileTime() {
```

Reply:
We already have a timestamp associated with the InProgress condition:
https://github.com/openshift/node-observability-operator/pull/52/files#diff-414059186bd44b4fc1458ec60a47e34426313d2470846d8e8d71a48d7f7506d3R90

Can that be used?

Reply:
Yeah, but I thought it will be helpful to have a separate parameter in case we need it for debugging, since nothing is logged if the cluster is in desired state.

Reply:
comment incorporated